src/Makefile.am

```diff
@@ -431,8 +431,8 @@ DISTCLEANFILES = obj/build.h
 
 EXTRA_DIST = leveldb
```

Comment:
You're missing the removal of leveldb here. This is unnecessary now that the files are enumerated in Makefile.leveldb.include